src/parse_async.cc

```diff
@@ -105,7 +105,7 @@ void AsyncParse(uv_work_t* request) {
     snowcrash::ParseResult<snowcrash::Blueprint> parseResult;
 
     // Parse the source data
-    drafter::ParseBlueprint(baton->sourceData, baton->options, parseResult);
+    drafter::ParseBlueprint(baton->sourceData, baton->options | snowcrash::ExportSourcemapOption, parseResult);
```

Comment:
`ExportSourcemapOption`. This was written long before the decision to move to `generate`. Our next major release which we are planning will rectify that.